packages/subscription-manager/src/components/UserSettings/UserSettings.tsx

```diff
@@ -13,7 +15,9 @@ type UserSettingsProps = {
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars -- until we start using any of these props
 const UserSettings = ( { value = {}, loading = false }: UserSettingsProps ) => (
-	<div className="user-settings">User settings will go here</div>
+	<div className="user-settings">
+		<BlockEmailsSetting onChange={ () => null } value={ false } />
```

Comment:
- The value prop can already be set with `value.block`/.
- The `onChange` prop can already be set with `(value) => ({block: value})` (the way value is obtained will depend on the implementation of the `FormInputCheckbox`).

Reply:
Cool, will change it 👍